### DISCUSSION TITLE
Is there a way to have a bang just do site: instead?

### DISCUSSION THREAD

**Discussion by Kvan7** [searxng/searxng#3158] [Q&A]
Basically just wondering if for my instance, could I have a bang do a normal search but as the site: filter thing. Specifically just for reddit as the reddit search is known as really bad, and I want to use a bang to save typing "site:reddit.com" and just do "!re". 
The other part of it is, when I do the bang version I get the images at the top which i really just don't want:
![image](https://github.com/searxng/searxng/assets/71402892/aac73f89-a493-4997-a945-9521bf36da46)
And doing site: gives just text results which i would much rather prefer when i do search reddit:
![image](https://github.com/searxng/searxng/assets/71402892/0cb267d3-0232-4a00-8480-9dbb278e2db1)

**Comment by Austin-Olacsi** [comment]
I have done this before, but for another website (not reddit). What I did:

1. copy one of the existing engines and rename it to something else
2. modify the engine to add site:example.com at the end of every search query
3. add the new engine to your searxng instance.

Then you can just use the bang for your new engine.

Also, that theme is epic. How did you use an image as the background?
